src/main/java/org/sagebionetworks/web/client/presenter/RejectReasonWidget.java

```diff
@@ -0,0 +1,151 @@
+package org.sagebionetworks.web.client.presenter;
+
+import com.google.gwt.user.client.rpc.AsyncCallback;
+import com.google.gwt.user.client.ui.IsWidget;
+import com.google.gwt.user.client.ui.Widget;
+import com.google.inject.Inject;
+import org.sagebionetworks.repo.model.UserProfile;
+import org.sagebionetworks.web.client.DisplayUtils;
+import org.sagebionetworks.web.client.utils.CallbackP;
+import org.sagebionetworks.web.client.widget.asynch.UserProfileAsyncHandler;
+import org.sagebionetworks.web.client.widget.entity.RejectReasonView;
+
+
+public class RejectReasonWidget implements RejectReasonView.Presenter, IsWidget {
+
+
+    // Template rejection format
+    public static String TEMPLATE_HEADER_HELLO = "Hi ";
+    public static String TEMPLATE_HEADER_THANKS = ",\n\nThank you for submitting your Synapse" +
+            " profile for validation. Before I can accept your request, you need to:\n";
+    public static String TEMPLATE_HEADER_SIGNATURE = "Let us know if you have any questions.\n\nRegards,\nact@sagebase.org";
+
+    // If no options are shown for rejected reason
+    public static String ERROR_MESSAGE = "Error: Please select at least one checkbox and generate a response or manually enter a response";
+
+    // Common reasons for user rejection
+    public static String REJECT_TAKE_SYNAPSE_QZ = "Take and pass the Synapse Certification quiz. ";
+    public static String REJECT_ADD_INFO = "Add at least one piece of information (education, employment, etc.) to your ORCID profile and set it to “public. ";
+    public static String REJECT_INITIAL_BOX = "Add at least one piece of information (education, employment, etc.) to your ORCID profile and set it to “public. ";
```

Comment:
Looks like REJECT_ADD_INFO is the same as REJECT_INITIAL_BOX.  Could you consolidate/fix, and also fix the typo (both an open quote without close quote on "public")